tests/recipes/cad1/task1/test_enhance.py

```diff
@@ -0,0 +1,150 @@
+"""Tests for the enhance module"""
+import numpy as np
+import torch
+from torchaudio.pipelines import HDEMUCS_HIGH_MUSDB
+
+from clarity.enhancer.compressor import Compressor
+from clarity.enhancer.nalr import NALR
+from recipes.cad1.task1.baseline.enhance import (
+    apply_baseline_ha,
+    decompose_signal,
+    get_device,
+    map_to_dict,
+    process_stems_for_listener,
+    separate_sources,
+)
+
+
+def test_map_to_dict():
+    sources = np.array([[1, 2], [3, 4], [5, 6]])
+    sources_list = ["a", "b", "c"]
+    output = map_to_dict(sources, sources_list)
+    expected_output = {
+        "left_a": 1,
+        "right_a": 2,
+        "left_b": 3,
+        "right_b": 4,
+        "left_c": 5,
+        "right_c": 6,
+    }
+    assert output == expected_output
+
+
+def test_decompose_signal():
+    np.random.seed(123456789)
+    # Load Separation Model
+    model = HDEMUCS_HIGH_MUSDB.get_model().double()
+    device = torch.device("cpu")
+    model.to(device)
+
+    # Create a mock signal to decompose
```

Comment:
Not for now, and I don't think this instance really requires it but when you want to mock things Python and `pytest` work well with the [Mock Library](https://realpython.com/python-mock-library/).